From 23d0380e1a41bca4be3c378acd174ec6d5cdd891 Mon Sep 17 00:00:00 2001
From: Steve Lhomme <robUx4@gmail.com>
Date: Mon, 3 Aug 2015 09:12:43 +0200
Subject: [PATCH 077/128] only use MINGW workarounds when not building with
 MSVC

--
it would be better to detect if we're using the Microsoft SDK or not
---
 include/vlc_windows_interfaces.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/vlc_windows_interfaces.h b/include/vlc_windows_interfaces.h
index e4c41f2..a9282c1 100644
--- a/include/vlc_windows_interfaces.h
+++ b/include/vlc_windows_interfaces.h
@@ -24,6 +24,8 @@
 #ifndef MINGW_WORKAROUNDS_H
 #define MINGW_WORKAROUNDS_H
 
+#ifndef _MSC_VER
+
 #ifdef __MINGW32__
 # include <_mingw.h>
 #endif
@@ -228,4 +230,5 @@ struct ITaskbarList3Vtbl
 # define THBN_CLICKED        0x1800
 #endif
 
+#endif /* _MSC_VER */
 #endif //MINGW_WORKAROUNDS_H
-- 
1.9.5.msysgit.1

